src/ui/flutter_app/lib/screens/settings/settings_card.dart

```diff
@@ -34,7 +35,13 @@ class SettingsCard extends StatelessWidget {
     return Card(
       color: AppTheme.cardColor,
       margin: AppTheme.cardMargin,
-      child: Column(children: children),
+      child: ListView.separated(
+        physics: const NeverScrollableScrollPhysics(),
+        shrinkWrap: true,
+        itemBuilder: (_, index) => children[index],
+        separatorBuilder: (_, __) => const ListItemDivider(),
```

Comment:
It looks like this technique was used so the above code heavily simplifies the middle separator.